integration/globalSetup.ts

```diff
@@ -0,0 +1,7 @@
+import { execSync } from "node:child_process";
+import { join } from "node:path";
+
+export default async function globalSetup() {
+  execSync("yarn build", { stdio: "inherit" });
+  execSync(join(__dirname, "scripts", "runCodegen.sh"), { stdio: "inherit" });
+}
```

Comment:
Instead of checking in the generated files, I'm generating this at test boot-up time.

Or do you like having the generated files in the repo for reference?

Reply:
For these sort of "the purpose of the utility is itself to generate code", I do like to keep the generated files checked in, b/c during code review, I almost look more at "how did the generated output change?" than "how did the codegen script itself change?" to get a gist for what is happening / provide some admittedly-human-based regression coverage.

Reply:
Sure thing - I'll un-ignore it :+1: